test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -219,7 +219,11 @@ public async Task CustomPropagator()
                             builder.ConfigureTestServices(services =>
                             {
                                 Sdk.SetDefaultTextMapPropagator(propagator.Object);
-                                this.tracerProvider = Sdk.CreateTracerProviderBuilder().AddAspNetCoreInstrumentation().AddInMemoryExporter(exportedItems).Build();
+                                this.tracerProvider = Sdk.CreateTracerProviderBuilder()
+                                                        .SetSampler(new TestSampler(SamplingDecision.RecordAndSample, new Dictionary<string, object> { { "SomeTag", "SomeKey" }, }))
```

Comment:
Would we still have a test for `CustomPropagator` which does not use a Sampler that adds tags?

If not, then make this test a `Theory` to which depending on a boolean value adds such a `Sampler` to the `CustomPropagator` test to cover both the scenarios.